flink-streaming-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumerBase.java

```diff
@@ -127,12 +125,7 @@
 	 */
 	public FlinkKafkaConsumerBase(List<String> topics, KeyedDeserializationSchema<T> deserializer) {
 		this.topics = checkNotNull(topics);
-		checkArgument(topics.size() > 0, "You have to define at least one topic.");
```

Comment:
I only just introduced the check in the previous commit, however it turned out hinder construction in tests. If you really like to keep the check around, i might come up with a different constructor that is used only for tests.